charts/temporal/templates/server-configmap.yaml

```diff
@@ -160,6 +161,7 @@ data:
           initialFailoverVersion: 1
           rpcName: "temporal-frontend"
           rpcAddress: "127.0.0.1:7233"
+          httpAddress: "127.0.0.1:7243"
```

Comment:
Do we need to worry about / can we generate a UUID for cluster ID here? Now that we are relying on it being present for https://github.com/temporalio/temporal/pull/6184

Reply:
The cluster ID is auto-generated.

```
$ temporal operator cluster describe --output json
{
  ...
  "clusterId": "855e5c4b-d994-4c72-a09b-3c18e5161bb2",
  ...
}

```